assets/src/edit-story/components/canvas/pagepreview/index.js

```diff
@@ -39,7 +39,8 @@ const Page = styled.button`
       isActive ? theme.colors.selection : theme.colors.bg.v1};
   height: ${({ height }) => height}px;
   width: ${({ width }) => width}px;
-  background-color: ${({ theme }) => theme.colors.mg.v1};
+  background-color: ${({ theme, backgroundColor }) =>
+    backgroundColor || theme.colors.mg.v1};
```

Comment:
Is it correct that the default bg color is `mg.v1` (gray) rather than `fg.v1` (white)? @miina?

Reply:
Since the editor displays a white background by default for the Page in designs then it should probably be white. The gray is likely taken from the default in the front-end. Should be changed to white most likely. cc @pbakaus for confirming.

Reply:
(This should then be reflected in FE as well.)

Reply:
no idea actually :) @samitron7 to confirm

Reply:
@samitron7 will get back Thursday on this topic.

Reply:
Previews now have white bg as default too, but generated stories keep their gray background until further notice. Created #424 to followup on this part.